Backend/router/v1/user.js

```diff
@@ -32,14 +35,18 @@ router.post('/signup', (req, res) => {
     }
   }).then(result => {
     if (result == "") {
-      models.user.create({
-        email: req.body.email,
-        password: hashedPassword,
-      }).then(result => {
-          res.send(
-            "< " + req.body.email + " > membership has been completed."
-          );
-      });
+      if (req.body.password.length < MIN_PASSWORD_LENGTH || req.body.password.length > MAX_PASSWORD_LENGTH ) {
+        res.status(400).send("Password must be between " + MIN_PASSWORD_LENGTH + ' and ' + MAX_PASSWORD_LENGTH + " characters long");
+      } else {
+        models.user.create({
+          email: req.body.email,
+          password: hashedPassword,
+        }).then(result => {
+            res.send(
+              "< " + req.body.email + " > membership has been completed."
```

Comment:
What does 'membership has been completed' mean? I don't think the message is clear.

Reply:
Can I change it like "Membership registration is complete" ?

Reply:
Yes, thank you.